Source/ThirdParty/Assimp/code/CMakeLists.txt

```diff
@@ -854,12 +858,12 @@ ADD_ASSIMP_EXPORTER( STEP
 )
 
 if ((NOT ASSIMP_NO_EXPORT) OR (NOT ASSIMP_EXPORTERS_ENABLED STREQUAL ""))
-	SET( Exporter_SRCS
-	  Common/Exporter.cpp
-	  CApi/AssimpCExport.cpp
-	  ${HEADER_PATH}/BlobIOSystem.h
-	)
-	SOURCE_GROUP( Exporter FILES ${Exporter_SRCS})
+    SET( Exporter_SRCS
```

Comment:
Try to minimize changes of the original files. Otherwise, you will have to repeat these changes with every upgrade. Style changes are not necessary.